solr/core/src/java/org/apache/solr/search/grouping/distributed/shardresultserializer/SearchGroupsResultTransformer.java

```diff
@@ -48,10 +48,9 @@ public SearchGroupsResultTransformer(SolrIndexSearcher searcher) {
   }
 
   @Override
-  @SuppressWarnings({"rawtypes"})
-  public NamedList transform(List<Command> data) throws IOException {
-    final NamedList<NamedList> result = new NamedList<>(data.size());
-    for (Command command : data) {
+  public NamedList<NamedList<Object>> transform(List<Command<?>> data) throws IOException {
+    final NamedList<NamedList<Object>> result = new NamedList<>(data.size());
+    for (Command<?> command : data) {
       final NamedList<Object> commandResult = new NamedList<>(2);
       if (SearchGroupsFieldCommand.class.isInstance(command)) {
```

Comment:
Maybe use `instanceof` here like in the `TopGroupsResultTransformer` class.
```suggestion
      if (command instanceof SearchGroupsFieldCommand) {
```